artifactory/v1/artifacts.go

```diff
@@ -183,3 +184,63 @@ func (s *ArtifactService) DeleteRepositoryReplicationConfig(ctx context.Context,
 	}
 	return s.client.Do(ctx, req, nil)
 }
+
+type Checksums struct {
+	Md5                    *string             `json:"md5,omitempty"`
+	Sha1                   *string             `json:"sha1,omitempty"`
+	Sha256                 *string             `json:"sha256,omitempty"`
+}
+
+type FileInfo struct {
+	Repo                   *string             `json:"repo,omitempty"`
+	Path                   *string             `json:"path,omitempty"`
+	Created                *string             `json:"created,omitempty"`
+	CreatedBy              *string             `json:"createdBy,omitempty"`
+	LastModified           *string             `json:"lastModified,omitempty"`
+	ModifiedBy             *string             `json:"modifiedBy,omitempty"`
+	LastUpdated            *string             `json:"lastUpdated,omitempty"`
+	DownloadUri            *string             `json:"downloadUri,omitempty"`
+	MimeType               *string             `json:"mimeType,omitempty"`
+	Size                   *int                `json:"size,string,omitempty"`
+	Checksums              *Checksums          `json:"checksums,omitempty"`
+	OriginalChecksums      *Checksums          `json:"originalChecksums,omitempty"`
+	Uri                    *string             `json:"uri,omitempty"`
+}
+
+// Description: Returns the metadata of the given file. Supported by local, local-cached and virtual repositories.
+// Security: Requires a privileged user (can be anonymous)
+func (s *ArtifactService) FileInfo(ctx context.Context, repoKey string, filePath string) (*FileInfo, *http.Response, error) {
+	path := fmt.Sprintf("/api/storage/%s/%s", repoKey, filePath)
+	req, err := s.client.NewRequest("GET", path, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	req.Header.Set("Accept", client.MediaTypeJson)
```

Comment:
I don't believe this is the correct type. You will want to create a constant in `types.go` and use `application/vnd.org.jfrog.artifactory.storage.FileInfo+json` for the value as per the [docs](https://www.jfrog.com/confluence/display/RTF/Artifactory+REST+API#ArtifactoryRESTAPI-FileInfo). This is the convention we've used elsewhere. This also allows you to get rid of the import for `"github.com/atlassian/go-artifactory/v2/artifactory/client"`

Reply:
Fixed with bb293b1